### ISSUE TITLE
Small issue

### ISSUE THREAD

**Issue by XxFri3ndlyxX** [Trsak/esx_inventoryhud#9]
So if you have a weapon and then go in your inventory and drag and drop to use. It will trow an error 

```
Error running system event handling function for resource es_extended: citizen:/scripting/lua/scheduler.lua:41: Failed to execute thread: @es_extended/server/main.lua:434: attempt to index a nil value
stack traceback:
        @es_extended/server/main.lua:434: in upvalue 'handler'
        citizen:/scripting/lua/scheduler.lua:219: in function <citizen:/scripting/lua/scheduler.lua:218>
stack traceback:
        [C]: in function 'error'
        citizen:/scripting/lua/scheduler.lua:41: in field 'CreateThreadNow'
        citizen:/scripting/lua/scheduler.lua:218: in function <citizen:/scripting/lua/scheduler.lua:182>
```

So if we check in extended its tells us 
```
RegisterServerEvent('esx:useItem')
AddEventHandler('esx:useItem', function(itemName)
	local xPlayer = ESX.GetPlayerFromId(source)
	local count   = xPlayer.getInventoryItem(itemName).count

	if count > 0 then
		ESX.UseItem(source, itemName)
	else
		TriggerClientEvent('esx:showNotification', xPlayer.source, _U('act_imp'))
	end
end)
```

So maybe add a check for weapon and to not allow to use :P But other than that works super fine

**Comment by bellane** [comment]
When I drop something it doesn't show an error but once I pick it back up it wont let me drop it again, it is gone from the inventory.

**Comment by Trsak** [comment]
Error fixed with 7c42dcf2a987eaa370842153f3d59db9ff6d89ad. Thanks for reporting.

@bellane 
Item dropping is handled by es_extended, so this is not inventory issue.
